tests/test_application_submission.py

```diff
@@ -183,3 +183,31 @@ def __init__(self, *k, **kwargs):
             # Try to save the application without any session data
             mock_request.POST = {}
             return ConfirmView().post(mock_request)
+
+    def test_application_success(self):
+        """
+        successfull response
+        :return:
+        """
+
+        with patch(
+            "request_a_govuk_domain.request.views.ConfirmView.save_application_to_database_and_send_confirmation_email"
+        ) as mock_save:
+            mock_save.return_value = True
+            res = self.client.post("/confirm/", data={})
+
+        self.assertEqual(302, res.status_code)
+
+    def test_application_failure(self):
+        """
+        failure response
```

Comment:
Same. Neither the infostring nor the title of the function explain that this test checks (maybe mention that it exists because of the double click problem for instance)